content/modules/redisgears/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: RedisGears
+description:
+weight: 55
+alwaysopen: false
+categories: ["Modules"]
+---
+## What is RedisGears?
+
+RedisGears is an engine for data processing in Redis. RedisGears supports batch and event-driven processing for Redis data. To use RedisGears, you write functions that describe how your data should be processed. You then submit this code to your Redis deployment for remote execution.
+
+As of v1.0.0, code for RedisGears must be written in Python. However, an internal C API exists and can be used by other Redis modules. In addition, support for other languages, including Java and JavaScript, is being planned.
+
+## Getting Started with RedisGears
+
+RedisGears is implemented by a Redis module. To use RedisGears, you'll need to make sure that your Redis deployment has the module installed. [Redis Cloud](https://redislabs.com/redis-enterprise-cloud/) and [Redis Enterprise](https://docs.redislabs.com/latest/rs/) both support the module natively. If you're running Redis Enterprise on-premises, see the [RedisGears installation docs]({{< relref "/modules/redisgears/redisgears-installation.md" >}}) for all the relevant details.
+
+If you're running open source Redis, you'll also need to [install the module](https://oss.redislabs.com/redisgears/quickstart.html) before using it.
+
+To get stated with RedisGears, see our [RedisGears Quick Start Tutorial]({{< relref "/modules/redisgears/redisgears-quickstart.md" >}}). To learn more about the RedisGears API and understand how it works under the hood, see of the [RedisGears docs](https://oss.redislabs.com/redisgears/).
+
+## Write-Behind Caching Patterns
+
+Redis users typically implement caching by using the look-aside pattern. However, with RedisGears, you can implement write-behind and read-through caching strategies, as well.
```

Comment:
where did you get the read-through from? @bmansheim

Reply:
@banker Added that. I think he just wrote it from scratch.